robot.py

```diff
@@ -130,10 +123,17 @@ def createObjects(self):
         self.joystick = wpilib.Joystick(0)
         self.gamepad = wpilib.XboxController(1)
 
-        self.spin_rate = 2.5
+        self.spin_rate = 1.5
 
     def autonomous(self):
         self.imu.resetHeading()
+        self.chassis.set_heading_sp(0)
+        self.hatch.enable_hatch = True
+        self.hatch_intake.alignment_speed = 0.5
+        self.hatch_deposit.alignment_speed = 0.5
+        for module in self.chassis.modules:
+            module.drive_motor.configVoltageCompSaturation(6, timeoutMs=10)
+            module.drive_motor.enableVoltageCompensation(True)
```

Comment:
Uhm........ don't you want this to not be here, and especially not to block the start of autonomous?

Reply:
We need to derate the motors at the beginning of autonomous, then bring them back up at the beginning of teleop. What would be your suggestion instead?

Reply:
It'd probably make sense to call a method in the chassis to do this instead of directly accessing attributes of the chassis here.

You definitely don't want to potentially take 40ms to do potentially nothing here though. Calling `enableVoltageCompensation` again is also unnecessary - we never disable it anywhere.